src/tools/kata-ctl/Cargo.toml

```diff
@@ -30,6 +30,15 @@ shim-interface = { path = "../../libs/shim-interface"}
 kata-types = { path = "../../libs/kata-types" }
 safe-path = { path = "../../libs/safe-path" }
 agent = { path = "../../runtime-rs/crates/agent"}
+vmm-sys-util = "0.11.0"
+epoll = "4.0.1"
+libc = "0.2.138"
+slog = "2.7.0"
+slog-scope = "4.4.0"
+hyper = "0.14.20"
+ttrpc = "0.6.0"
+tokio = "1.8.0"
+micro_http = { git = "https://github.com/firecracker-microvm/micro-http", branch = "main" }
```

Comment:
Move tests crates to `[dev-dependencies]`.

Reply:
fixed